File: src/main/java/de/tomjuri/armageddon/util/Logger.java
package de.tomjuri.armageddon.util;

import net.minecraft.util.ChatComponentText;

public class Logger {
    public static void info(Object message) { send("a" + message); }
    public static void error(Object message) { send("c" + message); }
    private static void send(String message) { Ref.player().addChatMessage(new ChatComponentText("§eArmageddon §8» §" + message)); }
}

File: src/main/kotlin/de/tomjuri/gemdigger/util/Logger.kt
package de.tomjuri.gemdigger.util

import net.minecraft.util.ChatComponentText

object Logger {
    fun info(message: Any) {
        send("a$message")
    }
    fun error(message: Any) {
        send("c$message")
    }
    private fun send(message: String) {
        player.addChatMessage(ChatComponentText("§cGem§eDigger §8» §$message"))
    }
}